consensus/event_loop.go

```diff
@@ -45,6 +45,7 @@ func (fc *fastHotStuffEventLoop) Init(
 	genesisQC QuorumCertificate,
 	tip BlockWithValidatorList,
 	safeBlocks []BlockWithValidatorList,
+	checkpointLatestView uint64,
```

Comment:
Please decouple the concept of a checkpoint from the event loop. May as well make this param the currentView the event loop initializes with so it's not another override
```suggestion
	currentView uint64,
```